### ISSUE TITLE
cdat using old libnetcdf

### ISSUE THREAD

**Issue by xylar** [E3SM-Project/e3sm-unified#18]
when I install the latest e3sm-unified (v 1.2.1), I see:
```
python -c "import vcs"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/xylar/test_e3sm_unified/base/envs/e3sm_unified_1.2.1_py2.7_nox/lib/python2.7/site-packages/vcs/__init__.py", line 102, in <module>
    from .utils import *  # noqa
  File "/home/xylar/test_e3sm_unified/base/envs/e3sm_unified_1.2.1_py2.7_nox/lib/python2.7/site-packages/vcs/utils.py", line 20, in <module>
    from . import boxfill
  File "/home/xylar/test_e3sm_unified/base/envs/e3sm_unified_1.2.1_py2.7_nox/lib/python2.7/site-packages/vcs/boxfill.py", line 20, in <module>
    from . import VCS_validation_functions
  File "/home/xylar/test_e3sm_unified/base/envs/e3sm_unified_1.2.1_py2.7_nox/lib/python2.7/site-packages/vcs/VCS_validation_functions.py", line 5, in <module>
    import genutil
  File "/home/xylar/test_e3sm_unified/base/envs/e3sm_unified_1.2.1_py2.7_nox/lib/python2.7/site-packages/genutil/__init__.py", line 5, in <module>
    from .grower import grower  # noqa
  File "/home/xylar/test_e3sm_unified/base/envs/e3sm_unified_1.2.1_py2.7_nox/lib/python2.7/site-packages/genutil/grower.py", line 3, in <module>
    import cdms2 as cdms
  File "/home/xylar/test_e3sm_unified/base/envs/e3sm_unified_1.2.1_py2.7_nox/lib/python2.7/site-packages/cdms2/__init__.py", line 33, in <module>
    from .dataset import createDataset, openDataset, useNetcdf3  # noqa
  File "/home/xylar/test_e3sm_unified/base/envs/e3sm_unified_1.2.1_py2.7_nox/lib/python2.7/site-packages/cdms2/dataset.py", line 7, in <module>
    from . import Cdunif
ImportError: libnetcdf.so.11: cannot open shared object file: No such file or directory
```
If I list the lib directory, the version actually present is 13, not 11:
```bash
ls /home/xylar/test_e3sm_unified/base/envs/e3sm_unified_1.2.1_py2.7_nox/lib/libnet*
/home/xylar/test_e3sm_unified/base/envs/e3sm_unified_1.2.1_py2.7_nox/lib/libnetcdf.a
/home/xylar/test_e3sm_unified/base/envs/e3sm_unified_1.2.1_py2.7_nox/lib/libnetcdff.a
/home/xylar/test_e3sm_unified/base/envs/e3sm_unified_1.2.1_py2.7_nox/lib/libnetcdff.so
/home/xylar/test_e3sm_unified/base/envs/e3sm_unified_1.2.1_py2.7_nox/lib/libnetcdff.so.6
/home/xylar/test_e3sm_unified/base/envs/e3sm_unified_1.2.1_py2.7_nox/lib/libnetcdff.so.6.1.1
/home/xylar/test_e3sm_unified/base/envs/e3sm_unified_1.2.1_py2.7_nox/lib/libnetcdf.settings
/home/xylar/test_e3sm_unified/base/envs/e3sm_unified_1.2.1_py2.7_nox/lib/libnetcdf.so
/home/xylar/test_e3sm_unified/base/envs/e3sm_unified_1.2.1_py2.7_nox/lib/libnetcdf.so.13
```

**Comment by xylar** [comment]
@doutriaux1, I'm hoping you can help me with this.  I suspect that the issue is that the current build of cdat and its dependencies is from 4 months ago and it is therefore using an outdated libnetcdf (not 4.6.1, as we require for the latest e3sm-unified).  Would is be easy for you to rebuild the cdat packages with the latest conda-forge bindings?

**Comment by xylar** [comment]
@doutriaux1, I am seeing the same issue on acme1 and aims4, so it really does seem to be an issue.  I'm not sure why it works on edison and cori, but my guess would be that libnetcdf.so.11 is in my library path somewhere, but not because of the anaconda environment.

**Comment by xylar** [comment]
I think I may have found a solution.  If I use cdms2 v. 3.0.1 instead of 3.0, it comes from conda-forge instead of cdat and it seems to work (at least the following works):

```
> conda create -n test_vcs -c cdat -c conda-forge python=2 vcs mesalib libnetcdf=4.6.1 hdf5=1.10.2
...
The following NEW packages will be INSTALLED:
...
    cdms2:                3.0.1-py27h6091dcd_1                  conda-forge
...

> conda activate test_vcs
> python -c "import vcs"
```

**Comment by xylar** [comment]
@doutriaux1, does that seem like the right solution to you?

**Comment by doutriaux1** [comment]
@xylar yes this is the way to go, we went through great pains to recompile everything on conda-forge with the newest compilers. You will need cdms2 3.0.1 and for the cdtime, libsdrs, libcdms, etc... Make sure you get the latest build of the latest version. We have nightly builds that tests these things so I'm confident it should work.
@zshaheen can help too, he's got a specific E3SM-DIAGS yml that dealt with tihs a week or so ago.

**Comment by xylar** [comment]
Thanks!  Let's move the discussion to #19.

**Comment by xylar** [comment]
addressed by #19
